tests/tests_pytorch/trainer/test_trainer.py

```diff
@@ -1107,6 +1107,7 @@ def val_dataloader(self):
         limit_val_batches=limit_val_batches,
         max_steps=1,
     )
+    trainer.fit(model)
     assert trainer.num_sanity_val_steps == num_sanity_val_steps
```

Comment:
@carmocca aren't num_sanity_val_steps already available before fit?

```suggestion
    assert trainer.num_sanity_val_steps == num_sanity_val_steps
    trainer.fit(model)
```

Reply:
You are right. I was confused and thought this was missing the `fit` call. Removed it and also the unused CustomModel class